### DISCUSSION TITLE
Feature request: System stability testing

### DISCUSSION THREAD

**Discussion by ZMJGMADHPXWT** [hardinfo2/hardinfo2#77] [Ideas]
Hello there.
I'd like to say thank you for such great software.
And the second thing I'd like to ask about system stability/stress testing implementation like Aida64 (for Windows) software has. It's loading CPU/MEM/Cache and creates temperature/throttling graphs.
Is that real?

Thanks in advance.

**Comment by hwspeedy** [comment]
Thanx for all of us.

I was going to say to use cpuburn, but that seams to have vanished from distros and is probably not enough any more - if not maintained - due to hardware changes.

Have a look at these CLI utils:
https://github.com/shibatch/cpuburnavx2
https://github.com/patrickmn/cpuburn

I don't think that cpu loading is for hardinfo2 - there is probably not one "cpuburner" for all scenarios...

Discussion is open for all with ideas....

**Comment by ZMJGMADHPXWT** [comment]
> Have a look at these CLI utils:

Haven't test them. But from the first glance 5 and 12 years old software doesn't look reliable. 

> cpu loading

But I'm talking not only about CPU load. I'd like to be able to test all the staff at the same time - memory, CPU, cache. And from my perspective hardinfo2 - is a right application to make a system stability tests.
